### PR TITLE
[MAJOR] Final review (first official release)

### PR DESCRIPTION
**Pull request by Vadorequest**
Left to do:
- [x] Use core.debug instead of console.log ?
- [x] Add unit tests (at least one to fix CodeClimate test coverage)
- [x] Check TODO in code (added during review)
- [x] Explain how this action is different from https://github.com/marketplace/actions/vercel-action

### CODE REVIEW

File: .github/workflows/run-example-deployment.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-18.04
     steps:
       - uses: actions/checkout@v2
-#        with:
+#        with: TODO what to do with this? Remove it?
 #          path: "github-action-deploy-on-vercel" # Act doesn't work if not specifying the path. See https://github.com/nektos/act/issues/228
```
Comment (Demmonius) [inline .github/workflows/run-example-deployment.yml]: Yes we can remove it because ACT is not implemented yet (and I guess it'll never)
```suggestion
```

**Review by Demmonius — CHANGES_REQUESTED** [review]

**Comment by Demmonius** [comment]
@Vadorequest may I have a review about the changes that I made please ? :)

File: lib/main.js [hunk 2]
```diff
@@ -67,7 +67,9 @@ const run = () => __awaiter(void 0, void 0, void 0, function* () {
 });
 runConfigChecks();
 run()
-    .then((actionReturn) => { })
+    .then((actionReturn) => {
+    core.debug(`actionReturn: ${actionReturn}`);
```
Comment (Vadorequest) [inline lib/main.js]: si c'est toujours du type string faut le préciser dans la signature

Comment (Demmonius) [reply lib/main.js]: Done ! The action doesn't return anything so I replaced it by a log

**Pull request merged**